client/webserver/site/src/js/doc.ts

```diff
@@ -291,6 +296,15 @@ export default class Doc {
     return fullPrecisionFormatter(prec).format(value)
   }
 
+  static decodeRateStep (rateStepEnc: number, baseUnitInfo: UnitInfo, quoteUnitInfo: UnitInfo) {
+    const [qFactor, bFactor] = [quoteUnitInfo.conventional.conversionFactor, baseUnitInfo.conventional.conversionFactor]
+    return rateStepEnc / (RateEncodingFactor * qFactor / bFactor)
+  }
```

Comment:
Maybe we could "decode" rate step in a simpler manner ? Just asking, in case I missed something.

Reply:
I wish this were easier.  I'm constantly forgetting that all asset's conversion factors are not 1e8 anymore.

Although I really don't know if this is the best order of operations.  Elsewhere we've found this was best: https://github.com/decred/dcrdex/pull/1733/files#diff-8466f141b8e33a7e492b165ae7b6509808343604c759f3a0cc36e06fc33f13efR892  https://github.com/decred/dcrdex/pull/1733#issuecomment-1303080513

Reply:
Should we also adjust `ConventionalRateAlt` then to do something like this ?
```
return float64(msgRate) * (float64(baseFactor) / float64(quoteFactor)) / RateEncodingFactor
```
It's doing this currently:
```
return float64(msgRate) / RateEncodingFactor * float64(baseFactor) / float64(quoteFactor)
```

Reply:
Not sure, but if we change the backend we should use big.Float for the math.